packages/edgeidentity/ios/src/RCTAEPEdgeIdentity.m

```diff
@@ -62,7 +62,12 @@ - (dispatch_queue_t)methodQueue
 RCT_EXPORT_METHOD(removeIdentity:(nonnull NSDictionary*)item
                   namespace:(NSString *)namespace) {
     
+    if (!item || !namespace) {
```

Comment:
With your latest changes, you need to move this below and check for !convertItem instead of item